### ISSUE TITLE
"mhlo.pad" error in MacOS when calling jax.numpy.cumsum

### ISSUE THREAD

**Issue by dkgaraujo** [google/jax#19402]
### Description

Details here:
https://github.com/keras-team/keras/issues/19059

### What jax/jaxlib version are you using?

jax v0.4.20 (from jax-metal)

### Which accelerator(s) are you using?

AMD GPU on MacOS

### Additional system info?

1.23.5 3.10.2 (v3.10.2:a58ebcc701, Jan 13 2022, 14:50:16) [Clang 13.0.0 (clang-1300.0.29.30)] uname_result(system='Darwin', node='mbp.lan', release='23.2.0', version='Darwin Kernel Version 23.2.0: Wed Nov 15 21:54:10 PST 2023; root:xnu-10002.61.3~2/RELEASE_X86_64', machine='x86_64')

**Comment by shuhand0** [comment]
The fix for supporting padOp with interior padding values will be in next release.
